src/blueapi/worker/task.py

```diff
@@ -10,9 +10,9 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class Task(BlueapiBaseModel):
+class TaskRequest(BlueapiBaseModel):
```

Comment:
This one I am unsure about, this class is definitely more than a request since it has a method to actually _run_ the task, but maybe it shouldn't? Maybe they should be separated? Thoughts welcome.

Reply:
Agreed, I think `do_task()` should be extracted somewhere else, could you make a separate issue for that please?